common/web/types/src/main-ldml-keyboard.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ *
+ * Just a wrapper to make `LdmlKeyboardTypes` export work.
+ */
+export { UnicodeSetParser, UnicodeSet } from './ldml-keyboard/unicodeset-parser-api.js';
+export { VariableParser, MarkerParser } from './ldml-keyboard/pattern-parser.js';
+export { ElementString } from './kmx/kmx-plus/element-string.js';
```

Comment:
Should this be separate from `LdmlKeyboardTypes`?

In #12712, I thought I saw `ElementString` exported separately from `LexicalModelTypes`.

Reply:
No, `ElementString` is not exported from `LexicalModelTypes`. There is just a unit test that refers to `ElementString` which was impacted by the update in #12712.

There's a bit of fuzziness between KMXPlus and LDMLKeyboard (because KMXPlus is an implementation for LDMLKeyboard spec) and I think for now, the layout I have is probably oaky.